pkg/build/builder/dockerutil.go

```diff
@@ -62,15 +73,35 @@ func pushImage(client DockerClient, name string, authConfig docker.AuthConfigura
 	if glog.V(5) {
 		opts.OutputStream = os.Stderr
 	}
-	var err error
+
 	var retriableError = false
 
+	eventWatch, err := eventClient.Watch(
```

Comment:
> not sure what you mean... a list gives you the latest resource version

Listing all events before a push doesn't seem like a great plan.  Is empty string treated as, "start watching now" or is it treated as zero?